### PR TITLE
Travis build status badge in README [skip ci]

### PR DESCRIPTION
**Pull request by reznord**
Adds build status badge to readme

### CODE REVIEW

**Comment by reznord** [comment]
Adding badges looks README colourful 😛 

@developit what other badges would you like to add here?

**Comment by SaraVieira** [comment]
@reznord About the npm downloads and version ?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# preact-cli [![Build Status](https://travis-ci.org/developit/preact-cli.svg?branch=master)](https://travis-ci.org/developit/preact-cli)
+# preact-cli [![Build Status](https://travis-ci.org/developit/preact-cli.svg?branch=master)](https://travis-ci.org/developit/preact-cli) [![NPM Downloads](https://img.shields.io/npm/dm/preact-cli.svg?style=flat)](https://www.npmjs.com/package/redux-form) [![NPM Version](https://img.shields.io/npm/v/preact-cli.svg?style=flat)](https://www.npmjs.com/package/preact-cli)
```
Comment (prateekbh) [inline README.md]: redux-form?

**Comment by reznord** [comment]
Fixed !, didn't see that earlier 😅

**Comment by reznord** [comment]
Any more badges that we can add?

**Comment by reznord** [comment]
Waiting for input from @developit @rkostrzewski

**Comment by developit** [comment]
Looks good! Just need to get those downloads up now haha

**Comment by reznord** [comment]
Hahaha, We sure will :)

_Sent from my OnePlus ONEPLUS A3003 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Pull request merged**